src/t_stream.c

```diff
@@ -2920,7 +2923,7 @@ void xclaimCommand(client *c) {
     }
 
     /* Do the actual claiming. */
-    streamConsumer *consumer = NULL;
+    streamConsumer *consumer = streamLookupConsumer(group,c->argv[3]->ptr,1);
```

Comment:
in the past we used to do the lookup (**and refresh**) only if `nack != raxNotFound`.
i'm not too familiar with this, please check if this is a bug fix or a bug.
@guybe7 ?

Reply:
mmm the old code was more correct but I can hardly say the new code is buggy: it just means that in the past if you tried to XCLAIM and you didn't actually claim anything the consumer seen_time wouldn't update. now it's always updated.. but anyway it's kind of an edge case (XCLAIM doing nothing)

Reply:
I think it is consistent with the past. I fixed it.